content/index.md

```diff
@@ -11,7 +11,7 @@ toc: false
 	<p>Fast 3kb alternative to React with the same ES6 API.</p>
 
 	<p>
-		<a href="/guide/getting-started" class="btn btn-primary btn-lg">Getting Started</a>
+		<a href="/guide/getting-started" class="white">Getting Started</a>
 		&nbsp; • &nbsp;
```

Comment:
@developit Something happened and now these spaces are missing in the webpage. Git diff is clean, weird.

It use to look like this:
https://cloud.githubusercontent.com/assets/1002461/18014137/2fdef6ca-6b87-11e6-953e-c949d92112dd.png

Now spaces around the dot are missing:
![screen shot 2016-09-19 at 09 17 26](https://cloud.githubusercontent.com/assets/1002461/18635235/f062a2be-7e49-11e6-8a1b-c74753fc8629.png)